public/client/search.js

```diff
@@ -7,17 +7,18 @@ function requestBook(bookId, pointCost, title, userPoints){
     }
     else{
         
-        modalBody.innerHTML = `Are you sure you want this book?: <br> <b>"${title}"</b>`
+        $("#requestTitle").text(title)
         var confirmButton = document.getElementById('confirmButton');
-        confirmButton.setAttribute("onclick", `request(${bookId}, ${pointCost}, '${title}')`);
+        //title was previously surrounded in single quotes
+        confirmButton.setAttribute("onclick", `request(${bookId}, ${pointCost}, "${title}")`);
```

Comment:
This button still shouldn't be done in the js. I noted this in the last pull request that the button should just be in the modal with an onclick as I don't think it's dynamic and that's how other buttons are created in our code base.